frontend/src/views/Evaluation/FollowUp/FollowUpView.tsx

```diff
@@ -88,15 +89,21 @@ const FollowUpView = ({ evaluation }: FollowUpViewProps) => {
                         return (
                             <div key={question.id}>
                                 <Divider />
-                                <QuestionAndAnswerFormWithApi
-                                    questionNumber={idx + 1}
-                                    question={question}
-                                    answer={answer}
-                                    disabled={disableAllUserInput || isParticipantCompleted}
-                                    onQuestionSummarySelected={onQuestionSummarySelected}
-                                    viewProgression={viewProgression}
-                                />
-                                <QuestionActionsListWithApi question={question} />
+                                <Box display="flex">
+                                    <Box>
```

Comment:
Maybe flexgrow here or is this expected behaviour?
![image](https://user-images.githubusercontent.com/25051255/105057758-d4a21e80-5a75-11eb-883d-b44dcd3a5a34.png)

```suggestion
                                    <Box flexGrow={1}>
```

Reply:
agree